tests/wamr-test-suites/wasi-test-script/run_wasi_tests.sh

```diff
@@ -14,6 +14,37 @@ readonly WAMR_DIR="${WORK_DIR}/../../../.."
 readonly IWASM_CMD="${WORK_DIR}/../../../../product-mini/platforms/${PLATFORM}/build/iwasm"
 readonly WAMRC_CMD="${WORK_DIR}/../../../../wamr-compiler/build/wamrc"
 
+run_aot_tests () {
+    tests=$1
+    for test_wasm in ${tests}; do
+        test_aot="${test_wasm%.wasm}.aot"
```

Comment:
Seems invalid? CI reports:
[Compiling -rwxr-xr-x to -rwxr-xr-x.aot](https://github.com/bytecodealliance/wasm-micro-runtime/actions/runs/4415242820/jobs/7738088476#step:12:440)
The wasm file name and aot file name are invalid